src/Stryker.Core/Stryker.Core/ProjectComponents/ProjectComponent.cs

```diff
@@ -44,37 +44,34 @@ public abstract class ProjectComponent : IReadOnlyInputComponent
         /// <summary>
         /// Returns the mutation score for this folder / file
         /// </summary>
-        /// <returns>decimal between 0 and 100 or null when no score could be calculated</returns>
-        public decimal? GetMutationScore()
+        /// <returns>decimal between 0 and 1 or null when no score could be calculated</returns>
+        public double GetMutationScore()
         {
-            var totalCount = TotalMutants.Count();
-            var killedCount = DetectedMutants.Count();
+            double totalCount = TotalMutants.Count();
+            double killedCount = DetectedMutants.Count();
 
-            if (totalCount > 0)
-            {
-                return killedCount / (decimal) totalCount * 100;
-            }
-            else
-            {
-                return null;
-            }
+            return killedCount / totalCount;
         }
 
         public Health CheckHealth(Threshold threshold)
         {
-            var mutationScore = GetMutationScore();
-
-            switch (mutationScore)
+            if (GetMutationScore() is var mutationScore && !double.IsNaN(mutationScore))
             {
-                case var score when score > threshold.High:
-                    return Health.Good;
-                case var score when score <= threshold.High && score > threshold.Low:
-                    return Health.Warning;
-                case var score when score <= threshold.Low && score > threshold.Break:
-                    return Health.Danger;
-                default:
-                    return Health.Danger;
+                var mutationScorePercentage = mutationScore * 100;
+
+                switch (mutationScorePercentage)
+                {
+                    case var score when score > threshold.High:
+                        return Health.Good;
+                    case var score when score <= threshold.High && score > threshold.Low:
+                        return Health.Warning;
+                    case var score when score <= threshold.Low:
```

Comment:
Why don't we make this the default? If the health is not good or warning it should be bad.